apps/checker/ping.go

```diff
@@ -65,6 +105,16 @@ func Ping(ctx context.Context, client *http.Client, inputData request.CheckerReq
 	}
 	defer response.Body.Close()
 
+	_, err = io.ReadAll(response.Body)
+	if err != nil {
```

Comment:
```suggestion
	if _, err := io.ReadAll(response.Body); err != nil {
```

Reply:
& not sure you need to decode the body to get the headers actually given that you don't do anything of the body.

Reply:
The endpoint can return some data, and to make sure the data is processable.